frontend/src/components/worksheets/WorksheetDialogs.js

```diff
@@ -199,6 +199,20 @@ class WorksheetDialogs extends React.Component {
                         </DialogContentText>
                     </DialogContent>
                 </Dialog>
+                {/* Schema saved message dialog */}
+                <Dialog
```

Comment:
Perhaps we should use the `Snackbar` component, which we use elsewhere for these kinds of messages?